pkgs/by-name/ve/ventoy/package.nix

```diff
@@ -28,14 +28,12 @@
 , withXfs ? false
 , withExt4 ? false
 , withNtfs ? false
-, withGtk3 ? false
-, withQt5 ? false
+, withGtk3 ? defaultGuiType == "gtk3"
+, withQt5 ? defaultGuiType == "qt5"
 , libsForQt5
 }:
 
 assert lib.elem defaultGuiType [ "" "gtk3" "qt5" ];
-assert defaultGuiType == "gtk3" -> withGtk3;
```

Comment:
The asserts should be kept, given that the user can possibly set the inputs in weird ways.